app/helpers/applications_helper.rb

```diff
@@ -24,6 +24,12 @@ def scraped_and_received_text(application)
     text
   end
 
+  def decode(description)
+    coder = HTMLEntities.new
+    first_decoding = coder.decode(description)
```

Comment:
@henare I had to call it twice to convert some entities. The first call converts &amp; to &, and the second call converts &#8217 to ', for example. If you know of better ways to do it or some insight to start, please let me know.